client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -214,7 +214,17 @@ public class RssSparkConfig {
   public static final ConfigEntry<Integer> RSS_CLIENT_ASSIGNMENT_RETRY_TIMES = createIntegerBuilder(
           new ConfigBuilder(SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_ASSIGNMENT_RETRY_TIMES))
           .createWithDefault(RssClientConfig.RSS_CLIENT_ASSIGNMENT_RETRY_TIMES_DEFAULT_VALUE);
-  
+
+  public static final ConfigEntry<Long> RSS_CLIENT_FALLBACK_RETRY_INTERVAL = createLongBuilder(
+      new ConfigBuilder("spark.rss.client.fallback.retry.interval")
```

Comment:
Could we use `spark.rss.client.access.retry.interval.ms`? Because we don't attempt to fallback, we want to access RSS.The variable's name should have unit, it improves readability.

Reply:
done.